res/database/Default/database_ads.xml

```diff
@@ -1031,7 +1033,7 @@ Only genuine with elk!</en>
 			</title>
 			<description>
 				<de>Knusprig knackiger Knabberspass mit feurig scharfer Chilinote. Nur für kurze Zeit mit Tiermotiven.</de>
-				<en>Crispy crunchy nibbles with fiery chillies. Only for a short time with animal patterns.</en>
+				<en>Crispy crunchy nibbles with fiery hot chili. Only for a short time with animal motifs.</en>
```

Comment:
Side note: prior you corrected "patterns" to "designs" : One wipe for kitchen AND toilet. Now with funny animal designs.

I think both fit (design and motifs). So just wanted to mention it in case you think one fits better.

Reply:
true, but using variations may also be fine.